src/aiida/common/pydantic.py

```diff
@@ -41,7 +41,7 @@ class Model(BaseModel):
     field_info = Field(default, **kwargs)
 
     for key, value in (('priority', priority), ('short_name', short_name), ('option_cls', option_cls)):
-        if value is not None:
+        if value is not None and field_info is not None:
```

Comment:
`Field` seems can return None, if so the `field_info` won't have `metadata` attribute.

Reply:
Nice fix. cc @edan-bainglass for visibility